doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -1782,13 +1782,73 @@ Example::
 Getting File Metadata
 ~~~~~~~~~~~~~~~~~~~~~
 
-Provides a json representation of the file metadata for an existing file where ``id`` is the database id of the file to replace or ``pid`` is the persistent id (DOI or Handle) of the file::
+Provides a json representation of the file metadata for an existing file where ``ID`` is the database id of the file to replace or ``PERSISTENT_ID`` is the persistent id (DOI or Handle) of the file
```

Comment:
@pkiraly it appears that "the file to replace" was already included the documentation text, but thought it was worth reviewing in your changes. Should this be changed to "the file to _get metadata from_" since this is not the file replace section?

Reply:
@mheppler I replaced the text.